lab-elizabeth/app/view/home/home-controller.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+require('./_home.scss');
+
+module.exports = ['$log', '$rootScope', 'galleryService', HomeController ];
+
+function HomeController($log, $rootScope, galleryService){
+  $log.debug('init homeCtrl');
+  this.galleries = [];
+
+  this.fetchGalleries = function(){
+    galleryService.fetchGalleries()
+    .then(galleries => {
+      this.galleries = galleries;
```

Comment:
Good job storing a reference to the service's gallery array on the scope of the home controller!